README.md

```diff
@@ -98,4 +104,17 @@ go get -u github.com/sourcegraph/src-cli/cmd/src
 3.  Travis will automatically perform the release. Once it has finished, **confirm that the curl commands fetch the latest version above**.
 4.  Update the `MinimumVersion` constant in the [src-cli package](https://github.com/sourcegraph/sourcegraph/tree/master/internal/src-cli/consts.go).
 
-**Note**: The version recommended by a Sourcegraph instance will be the highest patch version with the same major and minor version as the set minimum. This means that patch versions are reserved solely for non-breaking changes and minor bug fixes. This allows us to dynamically release fixes for older versions of src-cli without having to update the instance.
+### Patch releases
+
+The version recommended by a Sourcegraph instance will be the highest patch version with the same major and minor version as the set minimum. This means that patch versions are reserved solely for non-breaking changes and minor bug fixes. This allows us to dynamically release fixes for older versions of src-cli without having to update the instance.
+
+If you are releasing a bug fix or a new feature that is backwards compatible with one of the previous two minor version of Sourcegraph, the changes should be cherry-picked into a patch branch and re-releases with a new patch version. For example, suppose we have the the recommended versions.
+
+| Sourcegraph version | Recommended src-cli version |
+| ------------------- | --------------------------- | 
+| `3.x`               | `3.y.z`                     |
+| `3.{x-1}`           | `3.{y-1}.w`                 |
+
+If a new feature is added to a new `3.y.{z+1}` release of src-cli and this change requires only features available in Sourcegraph `3.{x-1}`, then this feature should also be present in a new `3.{y-1}.{w+1}` release of src-cli. Similar logic should follow for Sourcegraph `3.{x-2}` and its recommended version of src-cli.
```

Comment:
This might be easier to follow with specific numbers so the reader does not have to evaluate the version arithmetic at read-time.

For example if we use "3.104" and "3.105" for Sourcegraph versions and "3.68.4" and "3.67.18" for recommended versions, this becomes:

> If a new feature is added in the 3.68.5 release of `src-cli` and this change is compatible with Sourcegraph 3.104, then this feature **must** also be present in a new 3.67.19 release of `src-cli`. Similar logic applies for a release of Sourcegraph 3.103 and its recommended version of the `src-cli` tool

…or words to that effect. 

Also: In this suggestion I removed one of the two distinct uses of "feature" (the feature being released vs. features of the existing product) to avert confusion about which feature is the subject. 

I also replaced "should" with "must", since we are making here a rule for ourselves to follow during releases.